packages/dependency-extraction-webpack-plugin/index.js

```diff
@@ -130,37 +137,61 @@ class DependencyExtractionWebpackPlugin {
 
 				const runtimeChunk = entrypoint.getRuntimeChunk();
 
-				// Get a stable, stringified representation of the WordPress script asset.
-				const assetString = this.stringify( {
+				const assetData = {
 					dependencies: Array.from(
```

Comment:
This sorted part is the _stable_ bit from the comment below.

```suggestion
					// Get a sorted array so we can produce a stable, stringified representation.
					dependencies: Array.from(
```

We could consider moving all of this into `stringify`, it probably makes most sense there.